package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
```

Comment:
I don't think we should merge this change.

Reply:
Mean `package-lock.json` file ???